workers/unity/Assets/Playground/Config/PlayerTemplate.cs

```diff
@@ -3,13 +3,21 @@
 using Improbable.Gdk.Core;
 using Improbable.Gdk.PlayerLifecycle;
 using Improbable.Gdk.TransformSynchronization;
+using Improbable.PlayerLifecycle;
 
 namespace Playground
 {
     public static class PlayerTemplate
     {
-        public static EntityTemplate CreatePlayerEntityTemplate(string workerId, Improbable.Vector3f position)
+        public static EntityTemplate CreatePlayerEntityTemplate(string workerId, Improbable.Vector3f position,
+            Option<byte[]> playerCreationArguments)
```

Comment:
Why option? `byte[]` can be null

Reply:
Given that arguments are optional, it might be nicer for users to check `HasValue` rather than `== null`

Reply:
They feel like they are the same thing. The difference is that option is a non standard collection.